Producer/Producer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auxmoney\OpentracingAmqplibRabbitMqBundle\Producer;
+
+use Auxmoney\OpentracingBundle\Service\Tracing;
+use OldSound\RabbitMqBundle\RabbitMq\Producer as AmqplibProducer;
+use PhpAmqpLib\Channel\AMQPChannel;
+use PhpAmqpLib\Connection\AbstractConnection;
+use const OpenTracing\Tags\SPAN_KIND;
+use const OpenTracing\Tags\SPAN_KIND_MESSAGE_BUS_PRODUCER;
+
+/**
+ * Unfortunately, decorating the AmqplibProducer is not an option since other processes from the oldsound library
+ * rely on having a BaseAmqp class injected instead of using an interface
+ */
+final class Producer extends AmqplibProducer
+{
+    private const SPAN_NAME = 'Publishing message to "%s" exchange';
+    private const TAG_EXCHANGE_NAME = 'ExchangeName';
+    private const TAG_ROUTING_KEY = 'RoutingKey';
```

Comment:
We should follow the established naming conventions here, so I would suggest snake_case notation and a name like `message_bus.routing_key`.